Dockerfile.tool-reg

```diff
@@ -6,14 +6,13 @@ ENV PROGRAM=tool-registration
 
 COPY . /go/src/github.com/cyverse-de/permissions
 
-RUN go get github.com/constabulary/gb/...
-
-RUN cd /go/src/github.com/cyverse-de/permissions && \
-    gb build && \
-    cp bin/tool-registration /bin/
+RUN go install github.com/cyverse-de/permissions/...
 
 WORKDIR /
 
+ENTRYPOINT ["tool-registration"]
```

Comment:
I'm not quite ready to delete this conversion program yet. We may need it for sites who are running version 2.5.